docs/getting-started-guides/aws.md

```diff
@@ -19,6 +19,11 @@ Getting started on AWS EC2
 2. Install and configure [AWS Command Line Interface](http://aws.amazon.com/cli)
 3. You need an AWS [instance profile and role](http://docs.aws.amazon.com/IAM/latest/UserGuide/instance-profiles.html) with EC2 full access.
 
+NOTE: This script use 'default' AWS profile by default. You may explicitly set AWS profile to use using `AWS_DEFAULT_PFROFILE` environament variable:
```

Comment:
I think this should be "This script use**s the** 'default'..."

And also "using **the** `AWS_DEFAULT_PROFILE` ..."